Makefile

```
@@ -312,14 +318,20 @@ setup-dev: ##@setup Install all necessary tools for development
 setup-dev:
 	echo "Replaced by Nix shell. Use 'make shell' or just any target as-is."
 
-generate-handlers:
-	go generate ./cmd/generate_handlers/
-generate: ##@other Regenerate assets and other auto-generated stuff
-	go generate ./static ./static/mailserver_db_migrations ./t ./multiaccounts/... ./appdatabase/... ./protocol/... ./walletdatabase/... ./cmd/generate_handlers
-
-generate-appdatabase:
-	go generate ./appdatabase/...
-
+generate: PACKAGES ?= $$(go list -e ./... | grep -v "/contracts/")
+generate: GO_GENERATE_FAST ?= $(if $(shell which go-generate-fast),"true")
+generate: export GO_GENERATE_FAST_DEBUG ?= false
+generate: export GO_GENERATE_FAST_RECACHE ?= false
+generate:  ##@ Run generate for all given packages using go-generate-fast, fallback to `go generate` (e.g. for docker)
+	@GOROOT=$$(go env GOROOT); \
+	if [ "$(GO_GENERATE_FAST)" = "true" ]; then \
+		go-generate-fast -x $(PACKAGES); \
+	else \
+		go generate -x $(PACKAGES); \
+	fi
```

There is a simpler way to do this without spawning a separate shell to call `which`:
```suggestion
generate: PACKAGES ?= $$(go list -e ./... | grep -v "/contracts/")
generate: GO_GEN_CMD ?= $$(which go-generate-fast || echo 'go generate')
generate: export GO_GENERATE_FAST_DEBUG ?= false
generate: export GO_GENERATE_FAST_RECACHE ?= false
generate: ##@ Run generate for all given packages using go-generate-fast, fallback to `go generate` (e.g. for docker)
	GOROOT=$$(go env GOROOT) $(GO_GEN_CMD) -x $(PACKAGES)
```

---

noice!

---

Fixed, thank you!  ac38f50409b761d74709068f4b08634eeb5f631d